src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -397,7 +397,7 @@ private bool IsExpandWithinMultiLineLambda(SyntaxNode node)
 
                     return newToken;
                 }
-                catch (Exception e) when (FatalError.ReportUnlessCanceled(e))
+                catch (Exception e) when (!Renamer.isRunningInUnitTests && FatalError.ReportUnlessCanceled(e))
```

Comment:
Instead of this approach could we pass down a `Func<Exception, bool>` or an interface which abstracts away the `isRunningUnitTests` flag?